pkg/types/clusterconfig/clusterconfig.go

```diff
@@ -125,6 +127,20 @@ var UserValidation = &cr.StructValidation{
 				LessThanOrEqualTo:    pointer.Int64(16384),
 			},
 		},
+		{
+			StructField: "InstanceVolumeType",
+			StringPtrValidation: &cr.StringPtrValidation{
+				Default:   pointer.String("gp2"),
+				Validator: validateVolumeType,
+			},
+		},
+		{
+			StructField: "InstanceVolumeIops",
+			Int64PtrValidation: &cr.Int64PtrValidation{
+				Default:           pointer.Int64(3000),
```

Comment:
Yes, I thought about that too. I've chosen to set the default there because then you would require one less if statement in` clusterconfig.Validate()`.  But if you think it improves readability I can move it.

Reply:
Yes, I think it would be good to move it, mostly because then if the user specifies `iops` but is not using `io1`, then there will be an error with a nice message, which it more clear than if we silently ignore their iops configuration

Reply:
okay makes sense. Will do the changes later today